### PR TITLE
DOI overview card

### PR DESCRIPTION
**Pull request by viktor-yakubiv**
## Known bugs

- if the repository name is long, it will overlap with the number

### CODE REVIEW

**Comment by viktor-yakubiv** [comment]
@mcancellieri could you please check the design (in preview link), how we can get this data?

**Comment by mcancellieri** [comment]
I spoke with @valerapro about this. We are going to do an Elastic Search approach at first. And see how it looks like.

File: components/enrichment-chart/index.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        <LabelList
+          dataKey="caption"
+          position="insideLeft"
+          fill="#fff"
+          offset={16}
+        />
+        <LabelList
+          dataKey="value"
+          position="insideRight"
+          formatter={value => (+value).toLocaleString('en-GB')}
```
[the remark this answers is not in the document]
Comment (viktor-yakubiv) [reply components/enrichment-chart/index.jsx]: Just [checked](https://github.com/airbnb/javascript#coercion--numbers). Will do som thank you

**Comment by viktor-yakubiv** [comment]
I do not mind the animation.

**Pull request merged**